x-pack/plugins/transform/public/app/components/toast_notification_text.tsx

```diff
@@ -46,8 +51,9 @@ export const ToastNotificationText: FC<ToastNotificationTextProps> = ({ overlays
 
   const unformattedText = text.message ? text.message : text;
   const formattedText = typeof unformattedText === 'object' ? JSON.stringify(text, null, 2) : text;
-  const previewText = `${formattedText.substring(0, 140)}${
-    formattedText.length > 140 ? ' ...' : ''
+  const textLength = previewTextLength || 140;
```

Comment:
would be better to replace with `const textLength = previewTextLength ?? 140;`. currently presented code will override `0` value.